### ISSUE TITLE
Do some research (Google Scholar): choose which ML models to use

### ISSUE THREAD

**Issue by louisslnn** [louisslnn/ECSE343Project#6]
1st ideas:
- Linear Regression (classic, simple model)
- Random Forest (effective, random sampling type shi)
- SVR
- 1D Neural Network 

Explicit which frameworks will be used (Scikit-learn, PyTorch) for each chosen model

Train the models and use comparison metrics such as accuracy, training time, quantity of data needed, etc. Keep the one with the best metrics (best accuracy, lowest running time, etc.)
